src/main/java/org/minions/devfund/lourdes/MovieDatabase.java

```diff
@@ -0,0 +1,180 @@
+package org.minions.devfund.lourdes;
+
+import java.util.ArrayList;
+
+/**
+ * Class that manage the movie information.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * Constructor.
+     */
+    public MovieDatabase() {
+        this.movieList = new ArrayList<Movie>();
+        this.actorList = new ArrayList<Actor>();
+    }
+
+    /**
+     * Method to add a new movie to the movie list.
+     *
+     * @param name   represents the movie name.
+     * @param actors actor list from a movie.
+     */
+    public void addMovie(final String name, final String[] actors) {
+        Movie movie = createMovie(name);
+        ArrayList<Actor> movieActors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)